src/components/Snapshots/Snapshots.js

```diff
@@ -178,7 +177,7 @@ class Snapshots extends PureComponent {
           {isNotEmpty && (
             <Fragment>
               {' '}
-              <ExportButton handleClickExport={handleClickExport} timestamp={timestamp} />
+              <ExportButton handleClickExport={handleClickExport} />
```

Comment:
Why are we removing the `timestamp` from snapshots?

Reply:
It's just an unused prop